### ISSUE TITLE
Interface conversion error

### ISSUE THREAD

**Issue by xereeto** [jmhobbs/authy-cli#3]
Hi, I love this project cause it saved me from having to make my own! Now I'll never need to deal with that awful desktop application again.

Just FYI there's a problem related to interface conversion that shows up when you try to view a token.

`panic: interface conversion: interface {} is float64, not int`

I'm not hugely familiar with Go but the fix is simple, I just changed line 40 of model/token.go from

```go
iterations =t.KeyDerivationIterations.(int)
```
to
```go
iterations = int(t.KeyDerivationIterations.(float64))
```
I didn't want to submit a PR for such a small change so I figured I'd just open an issue first. Cheers :)

**Comment by jmhobbs** [comment]
Thanks for writing in @xereeto!  And I'm glad this tool has been of use to you.
I'll check this out and see if I can reproduce it, then I'll patch it up.

**Comment by xereeto** [comment]
Thanks :) I'm running go1.20.1 on M1 Mac, for reference.

**Comment by jmhobbs** [comment]
Fixed in 06800d3 and released in [v0.1.0](https://github.com/jmhobbs/authy-cli/releases/tag/v0.1.0), thanks!
